app/models/ability.rb

```diff
@@ -20,9 +20,5 @@ def initialize(user)
       can [:index], Vote
       can [:new, :create], VotePost
     end
-
-    user.permissions.each do |permission|
-      can permission.action.to_sym, permission.subject
-    end
```

Comment:
Before removing this, we must check that there are no actions in the regular namespace that require permissions other than those defined in `ability.rb`. Unfortunately, that is not the case. :cry:

Some old controllers, like `Contacts` or `Notices` still have admin actions located outside the admin namespace. Not even `:manage, :all` will let you modify those if you remove this block of code :disappointed: